docs/features/software-templates/index.md

```diff
@@ -12,7 +12,7 @@ code, template in some variables, and then publish the template to some
 locations like GitHub or GitLab.
 
 <video width="100%" height="100%" controls>
-  <source src="https://backstage.io/blog/assets/2020-08-05/feature.mp4" type="video/mp4" />
+  <source src="https://raw.githubusercontent.com/backstage/backstage/master/microsite/blog/assets/2020-08-05/feature.mp4" type="video/mp4" />
```

Comment:
We recently upgraded our Docusaurus version to v3, I'm not totally convinced this is the right direction forward for this. Can you look to see if we can accomplish this using static assets: https://docusaurus.io/docs/static-assets

Reply:
If the video needs to be used in both the blog and the docs, a static asset would be a better solution.